### ISSUE TITLE
eos_command cli transport cryptography error

### ISSUE THREAD

**Issue by phil-dileo** [ansible/ansible-modules-core#3884]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

eos_command
##### ANSIBLE VERSION

```
ansible 2.1.0.0
  config file = /Users/phil/ansible-demo/eos-ansible-quick-start/ansible.cfg
  configured module search path = Default w/o overrides
```
##### CONFIGURATION

Running under virtualenv.  The virtualenv was clean when I did a pip install ansible.
##### OS / ENVIRONMENT

Mac OS Yosemite - 10.10.3
##### SUMMARY

Trying to use eos_command with cli transport.  When you try to do this you get the paramiko error below.  The same command works over eapi.
##### STEPS TO REPRODUCE

Run any command via transport = cli using eos_command.

```

---
- hosts: eos_demo_group
  gather_facts: no
  connection: local

  tasks:
    - name: Arista EOS Base Configuration
      eos_command:
        commands='show version'
        provider={{ provider }}
```

##### EXPECTED RESULTS

I'd expect the command to be run, but we're not able to setup a connection to the switch.
##### ACTUAL RESULTS

```
TASK [Arista EOS Base Configuration] *******************************************
task path: /Users/phil/ansible-demo/eos-ansible-quick-start/run_commands.yaml:7
<dc1-spine1> ESTABLISH LOCAL CONNECTION FOR USER: phil
<dc1-spine1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo $HOME/.ansible/tmp/ansible-tmp-1465322773.59-161710036263092 `" && echo ansible-tmp-1465322773.59-161710036263092="` echo $HOME/.ansible/tmp/ansible-tmp-1465322773.59-161710036263092 `" ) && sleep 0'
<dc1-spine1> PUT /var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/tmpGtPaMW TO /Users/phil/.ansible/tmp/ansible-tmp-1465322773.59-161710036263092/eos_command
<dc1-spine1> EXEC /bin/sh -c 'LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8 python /Users/phil/.ansible/tmp/ansible-tmp-1465322773.59-161710036263092/eos_command; rm -rf "/Users/phil/.ansible/tmp/ansible-tmp-1465322773.59-161710036263092/" > /dev/null 2>&1 && sleep 0'
fatal: [dc1-spine1]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_name": "eos_command"}, "module_stderr": "No handlers could be found for logger \"paramiko.transport\"\nTraceback (most recent call last):\n  File \"/var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/ansible_Bb1ZiZ/ansible_module_eos_command.py\", line 181, in <module>\n    main()\n  File \"/var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/ansible_Bb1ZiZ/ansible_module_eos_command.py\", line 151, in main\n    response = module.execute(commands)\n  File \"/var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/ansible_Bb1ZiZ/ansible_modlib.zip/ansible/module_utils/eos.py\", line 239, in execute\n  File \"/var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/ansible_Bb1ZiZ/ansible_modlib.zip/ansible/module_utils/eos.py\", line 207, in connect\n  File \"/var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/ansible_Bb1ZiZ/ansible_modlib.zip/ansible/module_utils/eos.py\", line 155, in connect\n  File \"/var/folders/46/88fmnbd10qzfjrz_lxkh46dh0000gp/T/ansible_Bb1ZiZ/ansible_modlib.zip/ansible/module_utils/shell.py\", line 107, in open\n  File \"/Users/phil/.virtualenvs/ansible-demo/lib/python2.7/site-packages/paramiko/client.py\", line 338, in connect\n    t.start_client()\n  File \"/Users/phil/.virtualenvs/ansible-demo/lib/python2.7/site-packages/paramiko/transport.py\", line 493, in start_client\n    raise e\nImportError: dlopen(/Users/phil/.virtualenvs/ansible-demo/lib/python2.7/site-packages/cryptography/hazmat/bindings/_openssl.so, 2): Symbol not found: _SSL_CTX_get_ssl_method\n  Referenced from: /Users/phil/.virtualenvs/ansible-demo/lib/python2.7/site-packages/cryptography/hazmat/bindings/_openssl.so\n  Expected in: flat namespace\n in /Users/phil/.virtualenvs/ansible-demo/lib/python2.7/site-packages/cryptography/hazmat/bindings/_openssl.so\n", "module_stdout": "", "msg": "MODULE FAILURE", "parsed": false}
```

**Comment by phil-dileo** [comment]
@privateip have you seen this before? This seems to be a new issue since 2.1.0.0

**Comment by phil-dileo** [comment]
So I tested with paramiko==1.17.0 and it worked.  Not sure what happened in paramiko 2.0.0

**Comment by ansibot** [comment]
@privateip, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by gundalow** [comment]
Marking as blocking `2.2.0` as this may be a regression

**Comment by ansibot** [comment]
@privateip, @gundalow, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by privateip** [comment]
this is believed to be resolved, please reopen this issue if you are still experiencing the issue
